staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -77,6 +67,16 @@ type KubeSchedulerConfiguration struct {
 	// the default value (10s) will be used.
 	PodMaxBackoffSeconds *int64 `json:"podMaxBackoffSeconds,omitempty"`
 
+	// PercentageOfNodesToScore is the percentage of all nodes that once found feasible
```

Comment:
The field stays intact, so we're better not change its position in code.

Reply:
OK, will fix it.